java/jar/jar_test.go

```diff
@@ -196,27 +196,33 @@ func TestJAR(t *testing.T) {
 			ctx := zlog.Test(ctx, t)
 			f, err := td.Open(ent.Name())
 			if err != nil {
-				t.Fatal(err)
+				t.Error(err)
+				return
```

Comment:
I did this given the nature of the test being a during-development thing. I wanted the test to report errors but not stop iterating over the rest of the JARs.